pawn/Resources/Callbacks/Others/OnDialogResponse.pwn

```diff
@@ -297,7 +297,11 @@ deprecated_OnDialogResponse(playerid, dialogid, response, listitem, inputtext[])
                 return 1;
 
             switch(listitem) {
-                case 0: ShowPlayerDialog(playerid, DIALOG_COMMANDS_MAIN, DIALOG_STYLE_MSGBOX, "Main commands", "/help\r\n/stats\r\n/nos\r\n/getstats\r\n/world\r\n/my\r\n/minigames\r\n/jump\r\n/cd", "Back", "Exit");
+
+                case 0: ShowPlayerDialog(playerid, DIALOG_COMMANDS_MAIN, DIALOG_STYLE_MSGBOX, "Main commands", "/account\r\n/help\r\n/stats\r\n/nos\r\n/getstats\r\n/showmessage\r\n/world\r\n/my\r\n/minigames\r\n/jump\r\n/cd", "Back", "Exit");
+=======
```

Comment:
Would you mind fixing the merge conflict? Lines 301 and 303 roughly do the same, but we should only have one! 😛